code/firmware/rosco_m68k_firmware/stage2/romfs/include/romfs.h

```diff
@@ -0,0 +1,116 @@
+/*
+ *------------------------------------------------------------
+ *                                  ___ ___ _
+ *  ___ ___ ___ ___ ___       _____|  _| . | |_
+ * |  _| . |_ -|  _| . |     |     | . | . | '_|
+ * |_| |___|___|___|___|_____|_|_|_|___|___|_,_|
+ *                     |_____|       firmware v2
+ * ------------------------------------------------------------
+ * Copyright (c)2020-2024 Ross Bamford and contributors
+ * See top-level LICENSE.md for licence information.
+ *
+ * ROMFS filesystem interface
+ * 
+ * TODO this should move to librosco (in toolchain) once complete
+ * 
+ * ------------------------------------------------------------
+ */
+
+#ifndef __ROSCO_M68K_ROMFS_H
+#define __ROSCO_M68K_ROMFS_H
+
+#include <stdint.h>
+#include <stddef.h>
+#include <sys/types.h>
+
+#include "lfs.h"
+
+// TODO this is hardcoded to 360KB, needs to support 1440 (for 720KB) too...
+//
+// Should be linker-provided really...
+#ifndef ROMFS_BASE
+#define ROMFS_BASE          (((void*)(0x00E26000)))
+#endif
+
+#ifndef ROMFS_BLOCKSIZE
+#define ROMFS_BLOCKSIZE     ((512))
+#endif
```

Comment:
The blocksize for ROMFS staying at 512 could make a transition a bit trickier in the future if writing is added.

Reply:
Changed to 4096 👍

Reply:
I think that, in this case, it'll have to be 8192, since we've got two striped flash chips, each with a 4k sector size.

Reply:
Haha, just testing 😂 

Fixed, and fixed the block counts in the Makefile for building the ROMFS as well.